test/Microsoft.AspNetCore.SignalR.Redis.Tests/TestConnectionMultiplexer.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using StackExchange.Redis;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Tests
+{
+    public class TestConnectionMultiplexer : IConnectionMultiplexer
+    {
+        public string ClientName => throw new NotImplementedException();
+
+        public string Configuration => throw new NotImplementedException();
+
+        public int TimeoutMilliseconds => throw new NotImplementedException();
+
+        public long OperationCount => throw new NotImplementedException();
+
+        public bool PreserveAsyncOrder { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+
+        public bool IsConnected => true;
+
+        public bool IncludeDetailInExceptions { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public int StormLogThreshold { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+
+        // The event '---' is never used
+#pragma warning disable CS0067
```

Comment:
I'm not removing them from the interface, I'm shutting up the compiler because it complains about us not using them. When we do something with these in the `RedisHubLifetimeManager` then we can implement the behavior in the `TestConnectionMultiplexer` and remove the warning maybe.

Reply:
I understand that. I am saying the suppression should not be needed because we have to have these because they are part of the interface. We don't use them but we cannot remove because the code wouldn't compile. This is why I find it weird that we need to suppress warnings.

Reply:
OK. I think I [found it](https://blogs.msdn.microsoft.com/trevor/2008/08/14/c-warning-cs0067-the-event-event-is-never-used/). You need: 
```
public event EventHandler Unimportant
{
    add { }
    remove { }
}
```